app/models/resource.py

```diff
@@ -164,6 +164,174 @@ def generate_fake(count=20, center_lat=39.951021, center_long=-75.197243):
                 db.session.rollback()
 
     @staticmethod
+    def add_seattle_data():
+        from sqlalchemy.exc import IntegrityError
+        import os
+        import yaml
+
+        resources = os.listdir("_seattle")
+
+        description_descriptor = Descriptor(
+            name="description",
```

Comment:
nit: use single quotes for strings instead of double quotes (i.e., `'description'` instead of `"description"`)